cinema/serializers.py

```diff
@@ -18,13 +28,26 @@ class Meta:
 class CinemaHallSerializer(serializers.ModelSerializer):
     class Meta:
         model = CinemaHall
-        fields = ("id", "name", "rows", "seats_in_row", "capacity")
+        fields = (
+            "id",
+            "name",
+            "rows",
+            "seats_in_row",
+            "capacity"
```

Comment:
The `capacity` field in `CinemaHallSerializer` should be verified against the `CinemaHall` model to ensure it exists. If `capacity` is not a field in the model, this will cause an error.